src/test/ui/const-generics/min_const_generics/const-argument-if-length.rs

```diff
@@ -0,0 +1,24 @@
+// revisions: full min
```

Comment:
do these tests make sense in the `min_const_generics` folder, considering that they now also apply to `const_generics`?

Reply:
I was wondering that, but I'm not sure I see the value of having the `min_const_generics` folder at all, since almost every test applying to `min_const_generics` also applies to `const_generics` (except where we currently ICE for `const_generics`). Do you think it's useful to separate them like this, or should we merge the folders?

Reply:
The idea for the `min_const_generics` folder was to test the changes needed for `min_const_generics` when I started to implement it.

So I do think tests which explicitly check which param types are allowed and that only trivial expressions are allowed make some sense here.

But it would also be fine to just merge these tests with the remaining `const_generics` tests :shrug:

Reply:
I'll move these ones for now, and we can decide whether to merge the folders later.